doc/architecture/software-update.rst

```diff
@@ -35,7 +63,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
 signing the file containing the references to the hashes.
-A secure channel is not required to distribute the updates.
+There is no need for a secure channel, for distributing the updates.
```

Comment:
Are you saying that the swupd client is NOT using hash and signature checks to validate the received data?  Do we need to remove this paragraph, or what would we replace it with?